### DISCUSSION TITLE
In which episode was origin of the nickname "bagder" discussed?

### DISCUSSION THREAD

**Discussion by rybak** [fossified/podcast#48] [Q&A] [answered]
I think there was an episode of Fossified where it was discussed, but it might have been some other podcast where Daniel Stenberg appeared as a guess. Does anyone happen to remember?

**Comment by Eliot-Roxbergh** [comment]
IDK I was sure that it was one of the early Fossified episodes, like Pilot. Perhaps someone could suggest a free software transcribe software to automate the process of finding the bagder?

**Comment by bagder** [comment, marked as answer]
We talked about it roughly one minute into [epsiode one](https://pod.fossified.com/2023/03/09/s01e01.html) but I also have it [written down](https://daniel.haxx.se/about.html#bagder)
